UIKitDZ/UIKitDZ/ReservationViewController.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ReservationViewController.swift
+//  UIKitDZ
+//
+//  Created by Дмитрий Супрун on 22.09.22.
+//
+
+import UIKit
+/// Controller for make reservation
```

Comment:
```suggestion
/// Контроллер для создания брони
```

все описания на одном языке либо русский либо английский